beacon-chain/p2p/types/types.go

```diff
@@ -31,6 +32,17 @@ func (s *SSZUint64) MarshalSSZ() ([]byte, error) {
 	return marshalledObj, nil
 }
 
+// HashTreeRoot hashes the uint64 object following the SSZ standard.
+func (s *SSZUint64) HashTreeRoot() ([32]byte, error) {
```

Comment:
This is a generated file if im not mistaken. Are these generated code?

Reply:
These ones were hand made by @nisdas

Reply:
If i were to run `update-go-pbs.sh`, wouldn't these get overwritten?

Reply:
No, this isn't a PB thing.